Gems/Atom/Utils/Code/Source/PngFile.cpp

```diff
@@ -123,22 +206,31 @@ namespace AZ
             if (!end_info)
             {
                 png_destroy_read_struct(&png_ptr, &info_ptr, (png_infopp)NULL);
-                fclose(fp);
                 loadSettings.m_errorHandler("png_create_info_struct failed.");
                 return {};
             }
 
-AZ_PUSH_DISABLE_WARNING(4611, "-Wunknown-warning-option") // Disables "interaction between '_setjmp' and C++ object destruction is non-portable". See https://docs.microsoft.com/en-us/cpp/preprocessor/warning?view=msvc-160
+            AZ_PUSH_DISABLE_WARNING(
```

Comment:
super nit - if you really want it to look decent, leave the Macro whole in one line and set the comments above

Reply:
Fixed, this was auto-formatted by Clang when I cut and pasted the code. :(